package.json

```diff
@@ -4,75 +4,79 @@
     "description": "Learngroud React",
     "main": "index.js",
     "scripts": {
-        "start:dev": "webpack-dev-server --config webpack/webpack.dev.js --open --hot",
-        "build": "webpack --config webpack/webpack.prod.js",
-        "serve": "node server.js",
-        "start": "npm run build && npm run serve",
-        "test": "jest --coverage --coverageReporters=text-lcov | coveralls",
-        "test:dev": "jest"
+      "start:dev": "webpack-dev-server --config webpack/webpack.dev.js --open --hot",
+      "build": "webpack --config webpack/webpack.prod.js",
+      "serve": "node server.js",
+      "start": "npm run build && npm run serve",
+      "test": "jest --coverage --coverageReporters=text-lcov | coveralls",
+      "test:dev": "jest"
```

Comment:
Let us add `--coverage ` flag to `test:dev`, so that we can see the report locally.

Reply:
Ok. doing it right away

Reply:
For the  `--coverage` this will need us to add coverage file with repo token as part of environment variables. Do I go ahead and add them?